teuthology/repo_utils.py

```diff
@@ -70,33 +70,39 @@ def ls_remote(url, ref):
     return sha1
 
 
-def enforce_repo_state(repo_url, dest_path, branch, remove_on_error=True):
+def enforce_repo_state(repo_url, dest_path, branch, commit=None, remove_on_error=True):
     """
     Use git to either clone or update a given repo, forcing it to switch to the
     specified branch.
 
     :param repo_url:  The full URL to the repo (not including the branch)
     :param dest_path: The full path to the destination directory
     :param branch:    The branch.
+    :param commit:    The commit.
     :param remove:    Whether or not to remove dest_dir when an error occurs
```

Comment:
Hm.... not related to this PR but this should be updating according to actual signature of the function.